main.py

```diff
@@ -183,61 +183,104 @@ def main():
                     supported_currencies = get_all_currencies(single=True)
                 if supported_currencies:
                     if announcement_coin in supported_currencies:
-                        logger.debug("Starting get_last_price")
-                        price = get_last_price(announcement_coin, pairing)
+                        logger.debug("Starting get_coin_info")
+                        coin_info = get_coin_info(announcement_coin, pairing)
 
-                        logger.debug('Coin price: ' + price)
-                        logger.debug('Finished get_last_price')
+                        logger.debug('24h High: ' + coin_info.high_24h)
+                        logger.debug('Coin price: ' + coin_info.last)
+                        logger.debug('Finished get_coin_info')
 
                         try:
                             # Run a test trade if true
                             if config['TRADE_OPTIONS']['TEST']:
-                                order[announcement_coin] = {
-                                    'symbol': announcement_coin,
-                                    'price': price,
-                                    'volume': qty,
-                                    'time': datetime.timestamp(datetime.now()),
-                                    'tp': tp,
-                                    'sl': sl,
-                                    'id': 'test-order',
-                                    'text': 'test-order',
-                                    'create_time': datetime.timestamp(datetime.now()),
-                                    'update_time': datetime.timestamp(datetime.now()),
-                                    'currency_pair': f'{announcement_coin}_{pairing}',
-                                    'status': 'filled',
-                                    'type': 'limit',
-                                    'account': 'spot',
-                                    'side': 'buy',
-                                    'iceberg': '0'
-                                }
-                                logger.info('PLACING TEST ORDER')
-                                logger.debug(order[announcement_coin])
+                                if float(coin_info.high_24h) > float(coin_info.last) + (float(coin_info.last) * float(ath) / 100):
+                                    logger.info(f'24h High is {ath}% greater than price, coin not traded')
+                                else:
+                                    order[announcement_coin] = {
```

Comment:
Here is where the decision to not start a buy order on a new announced coin based on the % difference between current price and 24h high is done, also added to the logger info below.